Tasks/Gradle/CodeAnalysis/SonarQube/common.ts

```diff
@@ -47,9 +47,10 @@ export function processSonarQubeIntegration(sqBuildFolder:string):Q.Promise<void
     // Wait for all promises to complete before proceeding (even if one or more promises reject).
     return Q.allSettled([
```

Comment:
I'm leaving it unchanged in the interest of demonstrating how the function is intended to work.